scripts/algorithms/l2-svm.dml

```diff
@@ -144,10 +144,6 @@ while(continue == 1 & iter < maxiterations)  {
     s = be * s + g_new
     g_old = g_new
 
-	if(sum(s^2) == 0){
-	    continue = 0
-	}
```

Comment:
Also, if we're modifying these algorithms anyway, I would recommend to make continue and continue1 boolean variable and remove the respective branches.